src/libNode/DSBlockProcessing.cpp

```diff
@@ -405,9 +405,38 @@ void Node::StartFirstTxEpoch()
         }
     }
 
+    // Choose N other DS nodes to be recipient of microblock
+    m_DSMBReceivers.clear();
+    unsigned int numOfMBReceivers = NUM_MICROBLOCK_GOSSIP_RECEIVERS;
```

Comment:
Not critical, but for brevity and clarity I suggest in the future to use std::min, as I think this code pattern was used at least twice in this PR.